<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>1840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_topRow">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QScrollArea" name="linac0_ScrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>184</width>
          <height>1001</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="1" column="0">
          <widget class="PyDMTemplateRepeater" name="linac0">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="templateFilename" stdset="0">
            <string>cryomodule.ui</string>
           </property>
           <property name="dataSource" stdset="0">
            <string>linac0.json</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="PyDMLabel" name="PyDMLabel">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="accessibleName">
            <string>L0B</string>
           </property>
           <property name="text">
            <string>L0B</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="linac1_ScrollArea">
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>542</width>
          <height>1001</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0">
          <widget class="PyDMTemplateRepeater" name="linac1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="layoutType" stdset="0">
            <enum>PyDMTemplateRepeater::Horizontal</enum>
           </property>
           <property name="countShownInDesigner" stdset="0">
            <number>4</number>
           </property>
           <property name="templateFilename" stdset="0">
            <string>cryomodule.ui</string>
           </property>
           <property name="dataSource" stdset="0">
            <string>linac1.json</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="PyDMLabel" name="PyDMLabel_2">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="accessibleName">
            <string>L1B</string>
           </property>
           <property name="text">
            <string>L1B</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="linac2_ScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>216</width>
          <height>1001</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="1" column="0">
          <widget class="PyDMTemplateRepeater" name="linac2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="layoutType" stdset="0">
            <enum>PyDMTemplateRepeater::Horizontal</enum>
           </property>
           <property name="templateFilename" stdset="0">
            <string>cryomodule.ui</string>
           </property>
           <property name="dataSource" stdset="0">
            <string>linac2.json</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="PyDMLabel" name="PyDMLabel_3">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="accessibleName">
            <string>L2B</string>
           </property>
           <property name="text">
            <string>L2B</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QScrollArea" name="linac3_ScrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>980</width>
        <height>1001</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="PyDMTemplateRepeater" name="linac3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="layoutType" stdset="0">
          <enum>PyDMTemplateRepeater::Horizontal</enum>
         </property>
         <property name="templateFilename" stdset="0">
          <string>cryomodule.ui</string>
         </property>
         <property name="dataSource" stdset="0">
          <string>linac3.json</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="PyDMLabel" name="PyDMLabel_4">
         <property name="font">
          <font>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>L3B</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMTemplateRepeater</class>
   <extends>QFrame</extends>
   <header>pydm.widgets.template_repeater</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
